README.md

```diff
@@ -152,22 +152,22 @@ config files (and thus will not need to be specified).
 ### Commands for Students
 
 Students will mainly be concerned with submitting assignments and checking on the status of their submission.
-Therefore, the `autograder.cli.submission` package will be their primary resource.
+Therefore, the `autograder.cli.courses.assignments.submissions` package will be their primary resource.
```

Comment:
Here we will want to use `autograder.run`.
Since this section is about commands for students, and the shortcuts (`autograder.run.*`) should be all most students will need.

Reply:
```suggestion
Therefore, the `autograder.run` package will be their primary resource.
```